main.go

```diff
@@ -151,6 +142,15 @@ func runController(ctx context.Context, opts managerOpts) error {
 		setupLog.Error(err, "unable to create controller", "controller", "Domain")
 		os.Exit(1)
 	}
+	if err = (&controllers.TunnelReconciler{
+		Client:   mgr.GetClient(),
+		Log:      ctrl.Log.WithName("controllers").WithName("Tunnel"),
```

Comment:
super nit: technically the ingress controller is lower case with hyphens. Do you think we should make it human readable like this _tunnel_ -> _Tunnel_ ?

Reply:
I think the recorder is lower case with hyphens. The Ingress controller logs seems to just be:

```go
Log:            ctrl.Log.WithName("controllers").WithName("ingress"),
```

I'm fine with making the change to `Tunnel` -> `tunnel` and for `Domain` -> `domain` if you want